src/components/views/rooms/MemberList.js

```diff
@@ -163,14 +168,28 @@ module.exports = React.createClass({
         }
     },
 
-    onRoom: function(room) {
+    onRoom: async function(room) {
```

Comment:
No `await` added though?

Reply:
Indeed, had to code shifted around more before landing at this.